doogie_control/config/wheel_control.yaml

```diff
@@ -44,4 +44,23 @@ move_base_controller:
       has_acceleration_limits: true
       max_acceleration       : 1.5  # rad/s^2
       has_jerk_limits        : true
-      max_jerk               : 2.5  # rad/s^3
\ No newline at end of file
+      max_jerk               : 2.5  # rad/s^3
+
+# PID settings for class control_toobox::PID used in control loop of the Doogie Mouse wheels 
```

Comment:
Are you using this PID settings in move_base node?

Reply:
No, in the ros_control. This is used by this nodehandle: https://github.com/doogie-mouse/doogie_robot/blob/first_version/doogie_bringup/src/doogie_hardware.cpp#L47